peps/pep-0772.rst

```diff
@@ -0,0 +1,292 @@
+
+While we trust council members to act in the best interests of Python rather than themselves or their employers, the mere appearance of any one company dominating Python development could itself be harmful and erode trust.
+
+In a council election, if more than 2 of the top 5 vote-getters work for the same employer, then whichever of them ranked lowest is disqualified and the 6th-ranking candidate moves up into 5th place; this is repeated until a valid council is formed.
+
+During a council term, if changing circumstances cause this rule to be broken (for instance, due to a council member changing employment), then one or more council members must resign to remedy the issue, and the resulting vacancies can then be filled as normal.
+
+-------------------
+Packaging community
+-------------------
+
+Responsibility
+==============
+
+Packaging community members participate in formal votes to elect the Packaging Council.
+
+Processes
+=========
+
+Initial membership
+------------------
+
+Initial membership in the Packaging community will include anyone who has taken the time to formalise their participation in the Packaging community. This includes:
+
+* PyPA members: Anyone with the triage bit or commit bit, or at least one project in the PyPA organisation.
+* Packaging workgroup members: Anyone who is listed on the Packaging WG charter will be moved into the Packaging Community.
+* Interested core team members: Any Python core team member who is willing to participate is welcome.
+* Wider community members: Non-profit organisations that participate in packaging or working with new packagers. For example, PyOpenSci, NumFocus, Django, are encouraged to initially nominate up to 7 members by sending an email to \[todo\].
+
+Adding a new member
+-------------------
+
+Members are added to the Packaging community by a simple majority vote by the current membership. Quorum for adding new members is 50%.
+
+A vote to add a new member is triggered when a Packaging community member calls for one publicly on an appropriate communication channel, and another Packaging community member seconds the call within two weeks.
+
+Removal of a member
+-------------------
+
+In order to maintain a reasonable expectation of quorum, failure to participate in Packaging Council elections for two consecutive council elections automatically removes a person from the list of voting members, until they re-submit their intention to resume their participation to the Packaging Council in writing.
+
+In exceptional circumstances, it may be necessary to remove someone from the Packaging community against their will (for example: egregious and ongoing code of conduct violations). A Packaging community member may be removed by a two-thirds majority vote by the Packaging Council (in practice: 4:1 for a council with 5 individuals). If the ejected Packaging community member is also on the Packaging Council, then they are removed from the council as well.
```

Comment:
```suggestion
In exceptional circumstances, it may be necessary to remove someone from the Packaging community against their will (for example: egregious and ongoing code of conduct violations). A Packaging community member may be removed by a two-thirds majority vote by the Packaging Council (in practice: 4:1 for a council with 5 members). If the ejected Packaging community member is also on the Packaging Council, then they are removed from the council as well.
```

Reply:
What happens if this person is on the Packaging Council, do they get to participate in the vote?

Reply:
> What happens if this person is on the Packaging Council, do they get to participate in the vote?

Yes, and it's expected that they'd vote "no" and still get ejected FWIW.

Reply:
I'll clarify the language.